skops/hub_utils/_hf_hub.py

```diff
@@ -195,10 +202,19 @@ def _create_config(
     def recursively_default_dict() -> MutableMapping:
         return collections.defaultdict(recursively_default_dict)
 
+    if save_format == "auto":
+        extension = str(model_path).split(".")[-1]
```

Comment:
No change required, just fyi that `extension = Path(model_path).suffix` would also work.